src/v2/Components/Onboarding/Wizard.tsx

```diff
@@ -1,117 +1,35 @@
-  `/personalize/${CollectorIntentComponent.slug}`,
-  `/personalize/${Artists.slug}`,
-  `/personalize/${Genes.slug}`,
-  `/personalize/${BudgetComponent.slug}`,
-]
-
 export interface Props {
   redirectTo?: string
   tracking?: TrackingProp
 }
 
-export interface State {
-  finished: boolean
-}
-
-@track({}, { dispatch: data => Events.postEvent(data) })
-export class Wizard extends React.Component<Props, State> {
-  constructor(props) {
-    super(props)
-    this.state = {
-      finished: false,
-    }
-  }
-
-  onNextButtonPressed = (increaseBy: number, history) => {
-    history.push(STEPS[STEPS.indexOf(location.pathname) + increaseBy])
-  }
-
-  onFinish = () => {
-    this.setState({ finished: true })
-    setTimeout(() => (window.location.href = this.props.redirectTo || "/"), 500)
-
-    this.props.tracking.trackEvent({
-      action: "Completed Onboarding",
-    })
-  }
-
-  render() {
-    return (
-      <div>
-        <Route
-          path="/personalize/*"
-          render={() => (
-            <ProgressIndicator
-              percentComplete={
-                this.state.finished
-                  ? 1
-                  : STEPS.indexOf(location.pathname) / STEPS.length
-              }
-            />
-          )}
-        />
-
-        <Route
-          path={`/personalize/${CollectorIntentComponent.slug}`}
-          render={props => (
-            <CollectorIntent
-              {...props}
-              onNextButtonPressed={(increaseBy = 1) =>
-                this.onNextButtonPressed(increaseBy, props.history)
-              }
-            />
-          )}
-        />
-        <Route
-          path={`/personalize/${Artists.slug}`}
-          render={props => (
-            <Artists
-              {...props}
-              onNextButtonPressed={(increaseBy = 1) =>
-                this.onNextButtonPressed(increaseBy, props.history)
-              }
-            />
-          )}
-        />
-        <Route
-          path={`/personalize/${Genes.slug}`}
-          render={props => (
-            <Genes
-              {...props}
-              onNextButtonPressed={(increaseBy = 1) =>
-                this.onNextButtonPressed(increaseBy, props.history)
-              }
-            />
-          )}
-        />
-        <Route
-          path={`/personalize/${BudgetComponent.slug}`}
-          render={props => (
-            <Budget {...props} onNextButtonPressed={() => this.onFinish()} />
-          )}
-        />
-
-        {new RegExp(
-          `^/personalize(?!(/${CollectorIntentComponent.slug}|/${Artists.slug}|/${Genes.slug}|/${BudgetComponent.slug})).*$`
-        ).exec(location.pathname) && (
-          <Redirect to={`/personalize/${CollectorIntentComponent.slug}`} />
-        )}
-      </div>
-    )
-  }
+export const Wizard: React.FC<Props> = props => {
+  return (
+    <div>
+      <Route
+        path="/personalize/interests"
+        render={routeProps => <CollectorIntent {...routeProps} {...props} />}
+      />
```

Comment:
Mentioned in slack, but if possible we should swap out these React Router routes for found, which is the router we use everywhere else in the app. The basic setup in [these docs](https://github.com/4Catalyzer/found) nearly maps 1:1 with here, so it should be easy.

Reply:
Oh man but I _just_ learned about React Router, haha. I will do this migration in a follow up! 👍